util/logutil/log.go

```diff
@@ -260,6 +260,18 @@ func initFileLog(cfg *zaplog.FileLogConfig, logger *log.Logger) error {
 			return errors.New("can't use directory as log file name")
 		}
 	}
+
+	// Make sure we can open the log file the user has specified. lumberjack is going
+	// to manage this later, but we don't get any feedback about whether it could open
+	// the file or whether individual writes succeed.
+	f, err := os.OpenFile(cfg.Filename, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0644)
```

Comment:
should we also check the slow log file?

Reply:
Maybe we should. We should probably wrap log writes in something that checks errors, since we're using a library that automatically rotates the error log and might run into unknown problems writing, opening, closing, rotating at any time!

Reply:
Could we add this check to `pingcap/log`?